lib/video/screenRecording/desktop/firefoxrecorder.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const log = require('intel').getLogger('browsertime.video');
+const os = require('os');
+const util = require('util');
+const fs = require('fs');
+const path = require('path');
+const delay = ms => new Promise(res => setTimeout(res, ms));
+const execa = require('execa');
+const del = require('del');
+
+const unlink = util.promisify(fs.unlink);
+
+async function createTempFile() {
+  const mkdtemp = util.promisify(fs.mkdtemp);
+  const dir = await mkdtemp(path.join(os.tmpdir(), 'browsertime-'));
+  return path.join(dir, 'winrecorder.mp4');
+}
+
+async function enableWindowRecorder(enable, driver) {
+  if (enable) {
```

Comment:
I'd prefer to use my privileged JS bits, but no trouble: just `// TODO: ...` something here as a reminder.